.travis/config.sh

```diff
@@ -55,4 +48,17 @@ fi
 #   * .travis/runpytest.sh:
 #
 #       - RUN_PYTEST_SETUP_MODULE_UTILS
+#
+#   * .travis/runshellcheck.sh:
+#
+#       - RUN_SHELLCHECK_DISABLED
+#       - RUN_SHELLCHECK_EXTRA_ARGS
+type -f lsr_check_python_version > /dev/null 2>&1 || . ${SCRIPTDIR}/utils.sh
+if lsr_check_python_version python -lt 3.0
+then
+    PYTHON2_EXCLUDES="tests/ensure_provider_tests.py,scripts/print_all_options.py"
+    FLAKE8_DEFAULT_EXCLUDES=".svn,CVS,.bzr,.hg,.git,__pycache__,.tox,.eggs,*.egg"
+    RUN_PYLINT_EXCLUDE='^(\..*|ensure_provider_tests\.py|print_all_options\.py)$'
+    RUN_FLAKE8_EXTRA_ARGS="--exclude ${FLAKE8_DEFAULT_EXCLUDES},${PYTHON2_EXCLUDES}"
+fi
```

Comment:
This config code belongs to flake8/pylint but was moved to the shellcheck section in this PR.

Reply:
hmm - I was not aware that you were organizing the config.sh into "sections" . . .

Reply:
fixed in https://github.com/linux-system-roles/network/pull/261/commits/e2c5cd73388c6d75f3376f44aa5f2c799199c335